docs/changelog/0.11.0.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: 0.11.0
+---
+
+## Query API
```

Comment:
Should we add a written note about officially deprecating the `.search` and such?

Reply:
I'm on it